packages/aws-cdk-lib/aws-events-targets/lib/api-gatewayv2.ts

```diff
@@ -0,0 +1,73 @@
+import { ApiGatewayProps } from './api-gateway';
+import { addToDeadLetterQueueResourcePolicy, bindBaseTargetConfig, singletonEventRole } from './util';
+import * as apigwv2 from '../../aws-apigatewayv2';
+import * as events from '../../aws-events';
+import * as iam from '../../aws-iam';
+
+/**
+ * Use an API Gateway V2 HTTP APIs as a target for Amazon EventBridge rules.
+ */
+export class ApiGatewayV2 implements events.IRuleTarget {
+  private readonly _httpApi: apigwv2.IHttpApi;
+
+  /**
+   * @param httpApi - IHttpApi implementation to use as event target
+   * @param props - Properties to configure the APIGateway target
+   */
+  constructor(httpApi: apigwv2.IHttpApi, private readonly props?: ApiGatewayProps) {
+    this._httpApi = httpApi;
+  }
+
+  /**
+   * Returns the target IHttpApi
+   */
+  public get iHttpApi(): apigwv2.IHttpApi {
+    return this._httpApi;
+  }
+
+  /**
+   * Returns a RuleTarget that can be used to trigger this API Gateway HTTP APIs
+   * as a result from an EventBridge event.
+   *
+   * @see https://docs.aws.amazon.com/eventbridge/latest/userguide/resource-based-policies-eventbridge.html#sqs-permissions
+   */
+  public bind(rule: events.IRule, _id?: string): events.RuleTargetConfig {
+    if (this.props?.deadLetterQueue) {
+      addToDeadLetterQueueResourcePolicy(rule, this.props.deadLetterQueue);
+    }
+
+    const wildcardCountsInPath = this.props?.path?.match( /\*/g )?.length ?? 0;
+    if (wildcardCountsInPath !== (this.props?.pathParameterValues || []).length) {
+      throw new Error('The number of wildcards in the path does not match the number of path pathParameterValues.');
+    }
+
+    const apiArn = this._httpApi.arnForExecuteApi(
```

Comment:
nit: rename to follow the existing convention for [api-gateway.ts](https://github.com/aws/aws-cdk/blob/main/packages/aws-cdk-lib/aws-events-targets/lib/api-gateway.ts) and just so it's a bit more clear
```suggestion
    const httpApiArn = this._httpApi.arnForExecuteApi(
```

Reply:
Thanks, this var is updated